aldryn_forms/models.py

```diff
@@ -17,6 +19,7 @@
 from .helpers import is_form_element
 
 
+USES_TREEBEARD_ORDERING = LooseVersion(cms.__version__) >= LooseVersion('3.1')
```

Comment:
Can you not just check for the presence of Treebeard instead of relying on an indirect artifact?